bullet_train-api/app/helpers/api/open_api_helper.rb

```diff
@@ -46,6 +46,10 @@ def automatic_components_for(model, locals: {})
       path = "app/views/api/#{@version}"
       paths = [path, "app/views"] + gem_paths.product(%W[/#{path} /app/views]).map(&:join)
 
+      # Transform values the same way we do for Jbuilder templates
+      require "jbuilder/schema/template"
+      Jbuilder::Schema::Template.prepend ValuesTransformer
```

Comment:
I kinda think that this line should go in an initializer or something. I don't fully grok what it's doing, but I'm wondering if `jbuilder-schema` should just do this by default? (Apologies if that's a dumb question.)